content/applications/general/payment_acquirers.rst

```diff
@@ -38,31 +38,18 @@ trust.
    or Odoo databases hosted elsewhere. Instead, Odoo apps use a unique reference number to the data
    stored safely in the payment acquirers' systems.
 
-.. _payment_acquirers/acquirers:
+.. _payment_acquirers/supported_acquirers:
 
-Payment acquirers
-=================
+Supported payment acquirers
+===========================
 
-From an accounting perspective, we can distinguish two types of payment acquirers: the payments that
-go directly on the bank account and follow the usual reconciliation workflow, and the payment
-acquirers that are third-party services and require you to follow another accounting workflow.
+From an accounting perspective, we can distinguish two types of payment acquirers: the payment
+acquirers that are third-party services and require you to follow another accounting workflow, and
+the payments that go directly on the bank account and follow the usual reconciliation workflow.
```

Comment:
That's about it but it's really not clear as it is now. We should make another PR to clarify all of this soon. In the meanwhile, I'm merging this one because it only swaps content without modifying it.

Reply:
@AntoineVDV 
I beg to differ. My suggestion isn't the same.
previously "another accounting workflow" seemed a bit blurry, whereas "the payments go to the provider until you transfer them to your bank account" informs a lot more, imo.